config.example.yaml

```diff
@@ -32,6 +32,13 @@ templates:
             - mediawiki/v1/content
           # - mediawiki/v1/mobile
           # - mediawiki/v1/revision-scoring
+          modules:
+            - name: dummy-mod
+              version: 0.0.1
+              type: file
+              path: mods/dummy
```

Comment:
@gwicke Done. I put it directly in the modules branch, see [this commit](https://github.com/wikimedia/restbase/commit/bb964d2531de38a453934ffd9e2137a81c71a6c9). Also, this commit now allows one to drop the `path` stanza in case the module name matches its file name - convenient feature (TM).